src/pages/Details/Brief.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+
+export default class BriefInfo extends Component {
+  render() {
+    const {
+      id,
+      price,
+      size,
+      dscr,
+      degree,
+      expireDate,
+      keep,
+      category,
+      name,
+      companyInfo,
+      mainImage,
+      hashtag,
+    } = this.props;
+
+    return (
+      <div>
+        <div className="brief" key={id}>
```

Comment:
```suggestion
        <div className="brief">
```
key는 반복문이 사용되는 react 요소의 최상위 태그에만 사용하면 됩니다. 이 자리에서는 의미가 없을거 같아요

Reply:
확인 후 해당 사항 반영했습니다!